backend/Gemfile

```diff
@@ -0,0 +1,48 @@
+source "https://rubygems.org"
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby "3.1.0"
+
+# Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
+gem "rails", "~> 7.0.2", ">= 7.0.2.3"
+
+# Use sqlite3 as the database for Active Record
+gem "sqlite3", "~> 1.4"
+
+# Use the Puma web server [https://github.com/puma/puma]
+gem "puma", "~> 5.0"
```

Comment:
if you want you can upgrade to `5.6.2`, was a severe security vulnerability recently with rails+puma

https://github.com/advisories/GHSA-rmj8-8hhh-gv5h